scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java

```diff
@@ -20,10 +22,11 @@ public CouchbaseRepositoryFactory(CouchbaseSettings settings) {
 
     Cluster cluster = nodes.isEmpty() ? CouchbaseCluster.create() : CouchbaseCluster.create(nodes);
 
-    bucket =
+    bucket = Mono.fromCallable(() ->
         cluster
             .authenticate(settings.username(), settings.password())
-            .openBucket(settings.organizationsBucketName());
+            .openBucket(settings.organizationsBucketName())).retryBackoff(3, Duration.ofSeconds(1))

```

Comment:
I like infinity, why do you afraid of this endless?

Reply:
It's discussed solution, I you can explain why it's better to use another amount of retries, we can consider it.